files/fr/glossary/gutters/index.md

```diff
@@ -35,38 +31,38 @@ Dans l'exemple ci-dessous  nous avons une grille de pistes de 3 colonnes et 2 r
 ```css
 .wrapper {
   display: grid;
-  grid-template-columns: repeat(3,1.2fr);
+  grid-template-columns: repeat(3, 1.2fr);
   grid-auto-rows: 45%;
-  grid-column-gap: 20px;
-  grid-row-gap: 20px;
```

Comment:
The English example uses `grid-`, it is not necessary to remove it on the French version

Reply:
I noticed this on the English page, and the properties with a `grid-` prefix are actually considered obsolete, so they shouldn't be used — I currently have a PR in progress to make some small changes on the en-US side as well (cf. mdn/content#12759)